dotCMS/src/main/java/com/dotmarketing/util/starter/ExportStarterUtil.java

```diff
@@ -613,113 +423,32 @@ private List<FileEntry> getAdditionalDataAsJSON()  {
             contentAsJson = RulesImportExportUtil.getInstance().exportToJson();
             starterFiles.add(new FileEntry("RuleImportExportObject" + JSON_FILE_EXT, contentAsJson));
 
-            Logger.info(this, "Additional exportable JSON files have been generated successfully!");
+            Logger.debug(this, "Additional exportable JSON files have been generated successfully!");
         } catch (final Exception e) {
             Logger.error(this, e.getMessage(), e);
             throw new DotRuntimeException(e);
         }
         return starterFiles;
     }
 
-    private void moveAssetsToBackupDir() throws IOException{
-        String assetDir = ConfigUtils.getAbsoluteAssetsRootPath();
-
-        Logger.info(this, "Moving assets to back up directory: " + outputDirectory);
-
-        FileUtil.copyDirectory(assetDir, outputDirectory + File.separator + "assets", new AssetFileNameFilter());
-    }
-
     /**
      * Traverses the complete list of assets in the current dotCMS instance in order to retrieve their respective file
      * reference. This will allow dotCMS to add their contents to the final Starter ZIP file without copying them to
      * any backup or temporary location.
      *
      * @return The file references to all assets in the dotCMS repository.
      */
-    private List<FileEntry> getAssets() {
+    private void getAssets(final ZipOutputStream zip) {
         final String assetsRootPath = ConfigUtils.getAbsoluteAssetsRootPath();
-        final List<FileEntry> assetData = new ArrayList<>();
         final File source = new File(assetsRootPath);
-        Logger.info(this, "Adding Assets into the starter...");
-
-        FileUtil.listFilesRecursively(source, new AssetFileNameFilter())
```

Comment:
Would it be possible to unify this with the logic that is in the 
AssetExporterUtil?  We are using 2 different file filters to export the assets and I think we should only be using one.

Reply:
Yeah I can do that. I'll start working on it right away.

Reply:
Done!